doc/docs.md

```diff
@@ -69,6 +69,7 @@ you can do in V.
 </td><td valign=top>
 
 * [Advanced](#advanced)
+    * [Memory-unsafe code](#memory-unsafe-code)
     * [Calling C functions from V](#calling-c-functions-from-v)
```

Comment:
This should be put after `Calling C functions from V` since it has actual C calls in the examples.

Reply:
@medvednikov I've changed the example to use builtin `malloc` instead of `C.malloc`. The `Calling C` section will need to use `unsafe` blocks, so I think it's best to document `unsafe` first.